src/main/kotlin/commands/BanCommand.kt

```diff
@@ -0,0 +1,34 @@
+package commands
+
+import Command
+import ConfigManager.readConfigSafe
+import UserConfig
+import long
+import integer
+import string
+import doesLater
+import arg
+
+object BanCommand : Command("ec") {
+    init {
+        long("user") {
```

Comment:
what about @ mentions? 563138570953687061 is the user but `<@563138570953687061>` and `<@!563138570953687061>` is a ping. not to mention `living#0076` should look up that user and ban them. Take a look at purge command